src/styles/page.scss

```diff
@@ -0,0 +1,50 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: $block-gap;
+  padding: $calendar-padding;
+  width: width-calendar($day-size);
+
+  &__day {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    position: relative;
+    box-sizing: border-box;
```

Comment:
move this rule to `html` tag

Reply:
<img width="651" alt="image" src="https://github.com/mate-academy/layout_calendar/assets/70220667/f8acebac-8f53-4ee9-9b9a-9737618853fd">